package.json

```diff
@@ -44,6 +44,7 @@
     "lodash.includes": "^4.3.0",
     "lodash.set": "^4.3.2",
     "markdown-it": "~4.1.0",
+    "markdown-it-imsize": "^2.0.1",
```

Comment:
You must use `yarn add`/`yarn remove` to update package.json so that the yarn lockfile will be updated.

Reply:
Yeah, this was a PR from September, so I think was from before yarn on the OSF. But we updated the yarn lock in another PR, so unless somethings changed there, this should be better.